test/system/010-images.bats

```diff
@@ -110,4 +110,35 @@ Size        | [0-9]\\\+
     run_podman rm  mytinycontainer
 }
 
+@test "podman tag/untag" {
```

Comment:
@edsantiago, please take a look. My bash skills are limited but I want to follow your request to make sure we're extending the system tests.

Reply:
Greatly appreciated!